tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/decisiontree/ProbabilisticClassifier.scala

```diff
@@ -39,6 +38,20 @@ object ProbabilisticClassifier {
   implicit object ProbabilisticClassifierJsonFormat
       extends RootJsonFormat[ProbabilisticClassifier] {
 
+    // Override the Spray/JSON serialization for maps with integer keys, because these
+    // don't work out-of-the-box.
+    import spray.json.DefaultJsonProtocol.{ mapFormat => _, _ }
+    implicit val intMapFormat = new JsonFormat[Map[Int, Int]] {
```

Comment:
I'm actually kind of relieved to hear this, because I was wondering how that was supposed to work. :P